src/collection/dimensions.js

```diff
@@ -925,6 +951,21 @@ defineDimFns( {
   name: 'height'
 } );
 
+elesfn.padding = function(){
+  if(this.length > 0) {
+    var ele = this[0];
+    var _p = ele._private;
+    if( _p.autoPadding !== undefined ) {
```

Comment:
@d2fong 

I assume we're supporting percent padding only for compound parents.  That means we should check that the element is both a parent and it's padding is in percent units to use the autopadding.

We also need to note then that the percent padding is only for compounds in the docs.  I don't think it would be very useful for non-compound nodes, anyway.